json-c/meta.yaml

```diff
@@ -0,0 +1,33 @@
+package:
+    name: json-c
+    version: 0.12
+
+source:
+    git_url: https://github.com/json-c/json-c.git
+    git_tag: json-c-0.12-20140410
+
+build:
+    number: 0
+    skip: True  # [win]
+
+requirements:
+    build:
+        - autoconf
+        - automake
+        - libtool
+    run:
+        - libgcc
+
+test:
+    commands:
+        - exit $(test -f ${PREFIX}/lib/libjson-c.a)
+        - exit $(test -f ${PREFIX}/lib/libjson-c.so)
+
+about:
+    home: https://github.com/json-c/json-c
```

Comment:
In the Readme, they list the GitHub Wiki as the homepage ( https://github.com/json-c/json-c#building-on-unix-with-git-gcc-and-autotools ). Not sure it matters much, but figured it should be mentioned.